fetcher/package.json

```diff
@@ -47,9 +47,11 @@
     "@types/supertest": "^2.0.11",
     "@typescript-eslint/eslint-plugin": "^5.47.0",
     "@typescript-eslint/parser": "^5.47.0",
+    "@uniswap/v3-sdk": "^3.10.2",
```

Comment:
@KelvinThai Are we using this dependency for something? I couldn't pin point it anything imported in this PR 🤔